src/reuse/header.py

```diff
@@ -613,6 +619,11 @@ def add_arguments(parser: ArgumentParser) -> None:
         choices=list(NAME_STYLE_MAP),
         help=_("comment style to use, optional"),
     )
+    parser.add_argument(
+        "--fallback-dot-license",
+        action="store_true",
+        help=_("Write a .license file for unrecognised files"),
+    )
```

Comment:
Is it possible to use [ArgumentParser.add_mutually_exclusive_group](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group) for `--fallback-dot-license`, `--skip-unrecognised`, and `--force-dot-license`?